corehq/apps/reports/templates/reports/reportdata/data_export.html

```diff
@@ -2,6 +2,35 @@
 {% load hq_shared_tags %}
 {% load i18n %}
 
+{% block js %} {{ block.super }}
+    <script type="text/javascript" src="{% static 'reports/javascripts/reports.download_export.js' %}"></script>
+    <script type="text/javascript" src="{% static 'reports/ko/export.manager.js' %}"></script>
+{% endblock %}
+
+{% block js-inline %} {{ block.super }}
+    <script type="text/javascript">
+        var exportManager = new ExportManager({
+            bulk_download_notice_text: '{{ bulk_download_notice_text }}',
+            is_new_exporter: true,
+            bulkDownloadPageUrlRoot: '{{ download_page_url_root }}'
+        });
+        ko.applyBindings(exportManager, $('#export-ui').get(0));
+
+        var bulkExportTitle = "{% trans "What is Bulk Export?" %}";
+        var bulkExportContent = (
+            "{% trans "Bulk Export creates a single Excel file with each export as a separate sheet inside that file." %}"
+            + "<br /><br />{% trans "NOTE: Excel 2007+ is currently the only bulk export format supported." %}"
+        );
+        $('.bulk-export-info').popover({
```

Comment:
Super minor, but this seems like help text as much as info text & could use the same help pattern we use most other places (hq-help-template)